app/templates/components/permission-language.hbs

```diff
@@ -0,0 +1,18 @@
+{{yield}}
```

Comment:
There's no need to create a component for this. The markup should just go in preprint-footer-branded.hbs.